web-docs/walkthrough/index.adoc

```diff
@@ -0,0 +1,369 @@
+Here is a simple process in Python that you are welcome to use. 
+It takes an input filename and output file directory as arguments, copies the input file, and saves it as `new-output.txt` in your output file directory. 
+Name this Python file `input-output.py`.
+
+.input-output.py
+[source,python]
+----
+import os
+import sys
+
+# Demo program to input and output a file
+# Used to test Scale
+
+# Assign input filename
+INPUT_FILE = sys.argv[1]
+# Assign output directory 
+OUTPUT_DIR = sys.argv[2]
+# Assign output filename 
+OUTPUT_FILE = OUTPUT_DIR + '/new-output.txt'
+
+def main():
+    # Open input and output files.  Use of "with" will ensure proper closure of both files
+    # when with statements are exited.
+    with open(INPUT_FILE, mode='r') as infile:
+        with open(OUTPUT_FILE, mode='w') as outfile:
+            for line in infile:
+                outfile.write(line)
+
+# Call the main function
+main()
+----
+
+You will also need an input file in the right format for your process. 
+For this Python process, create a simple text file with a few lines of text, and name it with the `.txt` extension. 
+Save it in the same folder as your process.
+
+=== Download and Install the Seed Command Line Interface (CLI)
+
+You will find the latest version of Seed here: {seed-releases-url}
+
+* Scroll down and click on the green-outlined button on the left that says “Latest Release”
+* Scroll down to “Assets” and download the Seed version for your operating system. Note: “darwin” is the correct version for macOS.
+
+For more detailed instructions, refer to the Seed {seed-install-url}[installation instructions].
+
+You should now be able to run seed from any directory.
+
+=== Create a Dockerfile
+
+A Dockerfile is a set of instructions for what needs to be done to run your process. 
+Use a text editor to create it in the same directory as your process and name it simply “Dockerfile” (without any suffix).
+
+For a complete list of Dockerfile directives, see the {docker-reference-url}[Dockerfile reference].
+
+As an example, here is a Dockerfile to accompany the `input-output.py` file from Step 1.
+
+.Dockerfile
+[source]
+----
+FROM alpine:3.9
+
+RUN apk -U add python
+
+COPY ./input-output.py /app
+
+WORKDIR /app
+
+ENTRYPOINT ["python", "input-output.py"]
+----
+
+The Dockerfile defined in this example takes the following steps:
+
+* Start `FROM` the pre-existing `alpine:3.9` Linux image. This is an official image, validated by Docker.
+* `RUN` the command `apk -U add python` inside your image filesystem, which will install Python.
+* `COPY` the file input-output.py to your work directory.
+* Use `WORKDIR` to create the `/app` directory and specify that all subsequent actions should be taken from within `/app` _in your image filesystem_ (never the host’s filesystem).
+* Use `ENTRYPOINT` to describe how to run the container. In this case, it will run `python input-output.py`. The arguments will be specified at runtime.
```

Comment:
I'd say we omit use of entry point as we did in seed guide. The use of seed "command" should be preferred.

Reply:
I agree with you that it would be ideal to have this example match what's in the Seed user guide. 

That said, I simply took what Nikkala provided, converted it to AsciiDoc, made a few formatting changes, and incorporated it into the existing Scale docs.

I would suggest we merge this as-is, which will validate whether we've successfully integrated AsciiDoc into the existing documentation structure and add an issue to update the example to more closely match what's in the Seed user guide. 

With AsciiDoc successful integrated into the CI build, anyone should be able to update the docs going forward once they learn some AsciiDoc syntax, since they will have an example to work from and not need to worry about how the build works.

Thoughts? If you feel strongly we should update the example before merging this PR, I can update it or kick it back to Nikkala to have her make the changes.

Reply:
I'm in agreement. Let's create an issue to fix this after merge.

Reply:
Created issue [1890](https://github.com/ngageoint/scale/issues/1890)